db/pg/target-populate/city-council-districts.sql

```diff
@@ -0,0 +1,5 @@
+TRUNCATE 
+	city_council_district,
```

Comment:
The trailing comma causes a syntax error

```suggestion
	city_council_district
```